app/src/main/java/org/mozilla/fenix/settings/creditcards/view/CreditCardsManagementView.kt

```diff
@@ -23,12 +24,14 @@ class CreditCardsManagementView(
 ) : LayoutContainer {
 
     private val creditCardsAdapter = CreditCardsAdapter(interactor)
+    private val addCreditCardButtonAdapter = AddCreditCardButtonAdapter(interactor)
+    private val concatAdapter = ConcatAdapter(creditCardsAdapter, addCreditCardButtonAdapter)
```

Comment:
@Mugurell Can you double check this for me? This was the one approach I know of, but wanted to get a second pair of eyes.

Reply:
@gabrielluong Sorry for my late response.
ConcatAdapter would work but we need to keep in mind:
- this would mean the `Add credit card` button would scroll along the other items in the list. It can potentially be scrolled off screen. Not sure this is what we'd want.
- A11y needs special attention.

I would look into adding the button directly in `component_credit_cards` with something like https://gist.github.com/Mugurell/e5dec04adeb3c985ed0ca476335ec8be.
(instead of `include` a new "Add credit card" View can be declared) 
With the above we would have "Add credit card" immediately below the list of cards but never pushed to below the screen (`layout_constrainedHeight` is very important to be set so that the list will resize itself to not push the button to below the screen).

Reply:
@Mugurell Thanks for taking a look at this. 

@chrishernan It seems we still have quite of a work. Unfortunately, we're getting into a bit of a crunch time here in order to get the credit cards into the release by next Thursday. I am happy to have you continue working on this, but I also need to open this up for someone to carry this to completion in case you're busy. Sorry for any inconvenience and thank you for the patch!